### ISSUE TITLE
Apple TV support

### ISSUE THREAD

**Issue by Zegorax** [fredrikburmester/streamyfin#66]
Ciao, hope you're doing well!

Would it be possible to release it as an Apple TV bundle as well?

Thanks a lot in advance!

**Comment by retardgerman** [comment]
Hey!

Thanks for your idea! We are already working in the background to support more clients!

**Comment by 2600box** [comment]
Just wanted to say that if a barebones TestFlight build for AppleTV is possible, it would be highly appreciated.

**Comment by retardgerman** [comment]
Duplicate of #137

**Comment by Darknior** [comment]
I m dream about this feature too, with all my family we reading films on TV, my friends too ... and not on our phone :(
For now we can't use Streamyfin :(
And the official Jellyfin TV app is so boring and limited :(
Thanks for all your work <3

**Comment by Simon-Eklundh** [comment]
We're currently a bit blocked by some dependencies but are hoping to get it working in a while

**Comment by Darknior** [comment]
Thanks for your message :)
